examples/DKIM_gen_keys.phps

```diff
@@ -1,8 +1,8 @@
 <?php
 
 /**
- * This shows how to make a new public/private key pair suitable for use with DKIM.
- * You should only need to do this once, and the public key (**not** the private key!)
+ * This shows how to make a new public and private key pair suitable for use with DKIM.
```

Comment:
I think that's weaker and misleading. You might make "public and private keys", but not this.